contracts/src/systems/route/create_trade_route.cairo

```diff
@@ -0,0 +1,40 @@
+#[system]
+mod CreateTradeRoute{
+    use eternum::components::position::{Coord, Position};
+    use eternum::components::usage::Usage;
+    use eternum::components::trade::TradeRoute;
+    use traits::Into;
+
+    use dojo::world::Context;
+
+    fn execute(ctx: Context, from: Coord, to: Coord) {
```

Comment:
I would not create any Position component for the trade route entity. Just create an entity that has a TradeRoute component and a Usage component.
Trade routes should be indexed by start_coord and end_coord.